modules/update-upgrading-web.adoc

```diff
@@ -39,7 +47,7 @@ ifndef::openshift-origin[]
 +
 [IMPORTANT]
 ====
-For production clusters, you must subscribe to a `stable-\*`, `eus-*` or `fast-*` channel.
+For production clusters, you must subscribe to a `stable-\*`, `eus-*`, `fast-*`, `ga-*`, or `fleet-approved-*` channel.
```

Comment:
I recognize that the list has been made rather verbose, but I think stable and fast will be removed from this list in 4.16.